watertap/data/techno_economic/zeroth_order_model_data/decarbonator.yaml

```diff
@@ -0,0 +1,12 @@
+default:
```

Comment:
Example of unit model with no default performance data. If a particular case study was used to originally populate these parameters, I would eventually opt for choosing one of those case studies to provide default values here, with a metadata key pointing to that case study.

Reply:
In other words, I think it'd be better to include some values rather than provide no values and wait for the user to load data from case studies afterwards.